AutoRest/Generators/CSharp/CSharp/Templates/MethodTemplate.cshtml

```diff
@@ -30,17 +31,26 @@
 /// </param>
 public async Task<@(Model.OperationResponseReturnTypeString)> @(Model.Name)WithHttpMessagesAsync(@(Model.GetAsyncMethodParameterDeclaration(true)))
 {
+    @foreach (var groupParameter in Model.ParameterTemplateModels.Where(p => p.IsParameterGroup))
```

Comment:
make sure you adjust the generated comments for group parameters as well, do the same for node.js and java as well.